### DISCUSSION TITLE
[Config Support]: Object detection only detecting "person"

### DISCUSSION THREAD

**Discussion by Hexadecimalism** [blakeblackshear/frigate#13523] [Config Support] [answered]
### Describe the problem you are having

My setup only seems to detect "person". I'm not sure what I've done wrong, I know "- person" is the default, but it seems to ignore the other flags I've included. The full recording captures tons of footage of people, chickens, other birds, dogs, cats, foxes and the occasional badger. Along with not  showing events and clips for everything but people, I should expect foxes and badgers to also be identified (albeit misclassified foxes as cats/dogs and badgers as bears maybe). Any idea what I've done wrong based on my config file?

### Version

0.13.2-

### Frigate config file

```yaml
mqtt:
  enabled: false # set to "true" once everything is working properly to enable image pushing integrations
detectors:
  coral:
    type: edgetpu
    device: usb
record:
  enabled: true
  retain:
    days: 31
    mode: all
  events:
    retain:
      default: 365
      mode: all # my default is "motion", testing all
birdseye:
  enabled: true
  mode: continuous
  restream: true
cameras:
  Chickens: # Camera Name
    birdseye:
      order: 1
    ffmpeg:
      hwaccel_args: preset-vaapi
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps
          1 -strftime 1 -c:v copy -tag:v hvc1 -bsf:v hevc_mp4toannexb -c:a aac
        rtmp: -c:v copy -c:a aac -f flv
      inputs:
      - path: rtsp://user:pass@address:port/1   # 4k Stream
        roles:
        - record
        - rtmp
        - audio
      - path: rtsp://user:pass@address:port/2   # 720p Stream
        roles:
        - detect
    rtmp:
      enabled: false # RTMP should be disabled (false) if your stream is not H264 (These streams are H265, but can be changed to H264)
    detect:
      enabled: true
      fps: 5
      width: 1280 # <- optional, by default Frigate tries to automatically detect resolution
      height: 720 # <- optional, by default Frigate tries to automatically detect resolution
      min_initialized: 2
      max_disappeared: 25 # consider 30 (note, detect FPS is 5)
    objects:
      track:
      - person
      - dog
      - cat
      - bird
      - bicycle
      - horse
      - sheep
      - bear
      - cow
      - elephant
      - giraffe
    snapshots:
      enabled: true
      timestamp: false
      bounding_box: true
      retain:
        default: 31
    motion:
      mask:
      - 0,0,0,350,174,344,262,128,648,100,900,113,1022,108,1280,289,1280,0
```

### Relevant Frigate log output

```shell
[INFO] Preparing Frigate...
[INFO] Starting Frigate...
[2024-09-03 17:34:59] frigate.app                    INFO    : Starting Frigate (0.13.2-)
[2024-09-03 17:34:59] peewee_migrate.logs            INFO    : Starting migrations
[2024-09-03 17:34:59] peewee_migrate.logs            INFO    : There is nothing to migrate
[2024-09-03 17:34:59] frigate.app                    INFO    : Recording process started: 2237523
[2024-09-03 17:34:59] frigate.app                    INFO    : go2rtc process pid: 141
[2024-09-03 17:34:59] frigate.app                    INFO    : Output process started: 2237536
[2024-09-03 17:34:59] detector.coral                 INFO    : Starting detection process: 2237533
[2024-09-03 17:34:59] frigate.app                    INFO    : Camera processor started for Chickens: 2237546
[2024-09-03 17:34:59] frigate.app                    INFO    : Capture process started for Chickens: 2237548
[2024-09-03 17:34:59] frigate.detectors.plugins.edgetpu_tfl INFO    : Attempting to load TPU as usb
[2024-09-03 17:35:01] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found
```

### Relevant go2rtc log output

```shell
NA
```

### Operating system

Proxmox

### Install method

Docker Compose

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

![image](https://github.com/user-attachments/assets/74962c0e-a186-46b1-9138-d3d3aabaf95d)

### Any other information that may be helpful

My installation is an [LXC](https://tteck.github.io/Proxmox/#frigate-lxc) (though I doubt it is relevant unless my install is out of date).

**Comment by NickM-27** [comment]
it's possible that the objects are not detected with enough confidence. I would suggest using the debug view to see what is detected

**Reply by Hexadecimalism** [reply]
While a possible avenue, it was not the cause of my issue, I'll consider this when I start using custom models.

**Comment by Hexadecimalism** [comment, marked as answer]
Problem was not Frigate. I was using the lower resolution substream on my camera for detection, but it seems like the lower resolution stream sends the same image for a few hours without updating, followed by a few more hours of updating, repeat ad nauseum. The higher resolution stream works perfectly the entire time. Rebooting the camera solves the issue, but it looks like a firmware issue. My current solve is having the camera reboot every 6 hours which isn't very sensible.
On paper I could use my primary stream twice as detect stream as well, just force the detect resolution like I always do. Though I wonder if that loses or gains clarity compared to the actual 720p stream.

I'd like to keep the thread open for discussion, but my issue is technically solved, as it's not a frigate issue per se, so I'll close it.
